server-common/src/test/java/org/apache/kafka/server/util/timer/SystemTimerReaperTest.java

```diff
@@ -59,4 +59,11 @@ public void testReaper() throws Exception {
             timer.close();
         }
     }
+
+    @Test
+    public void testReaperClose() throws Exception {
```

Comment:
It seems to me the test should check two conditions after calling `close`:

1. `SystemTimerReaper` is not running
2. `SystemTimer#close` happens once

Reply:
I've addressed the comments in the latest commit, many thanks !